logic.py

```diff
@@ -1415,6 +1417,33 @@ def subst(s, x):
     else:
         return Expr(x.op, *[subst(s, arg) for arg in x.args])
 
+def cascade_substitution(s):
+    '''This method allow to return a correct unifier in normal form
+    and perform a cascade substitution to s.
+    For every mapping in s perform a cascade substitution on s.get(x)
+    and if it is replaced with a function ensure that all the function 
+    terms are correctly updates by passing over them again.
```

Comment:
It should either be 'correct updates' or 'correctly updated'.

Reply:
I reviewed all the requests and committed the relative commit to fix them but I don't understand why The Travis CI build failed

Reply:
All checks have passed in the new commit! It should be all right now.